RealmSwift-swift2.0/Object.swift

```diff
@@ -221,6 +204,35 @@ public class Object: RLMObjectBase {
         }
     }
 
+    // MARK: Equatable
+
+    /// Returns whether both objects are equal.
+    /// Objects are considered equal when they are both from the same Realm
+    /// and point to the same underlying object in the database.
+    public override func isEqual(object: AnyObject?) -> Bool {
```

Comment:
Yes, but we're only adding it for Equatable conformance, so the mark comment is still appropriate IMO.